README.md

```diff
@@ -184,12 +184,10 @@ test uses the Web3 node.js library as a client to run tests against fab3 and the
 EVMCC.
 
 ## Contributions
-The `fabric-chaincode-evm` lives in a [gerrit repository](https://gerrit.hyperledger.org/r/#/admin/projects/fabric-chaincode-evm).
-The github repository is a mirror. For more information on how to contribute
-look at [Fabric's CONTRIBUTING documentation](http://hyperledger-fabric.readthedocs.io/en/latest/CONTRIBUTING.html).
+Pull requests are now accepted on github. When submitting a pull request, please
+include the JIRA number associated with your JIRA work item in the commit message.
```

Comment:
"commit message title" or "commit message subject line" I think we still want it to be in the first line for the script (which we should rewrite so it can find anywhere in the body, but that's a different subject)

Reply:
updated